src/components/containers/LectureCard/chips/LocationChip.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LocationIcon from '../../../../assets/images/icons/location.svg';
```

Comment:
../../../../../../../../../../bruh.jpg

Is there any plugin that can replace it with something like `...from @assets/images/bruh.jpg`?